prompt_poet/prompt.py

```diff
@@ -451,6 +454,13 @@ def _render_parts(self):
                     "Tokens were provided in template. Regular tokenization will be skipped."
                 )
                 self._total_tokens += len(part.tokens)
+    
+    def _validate_template_replacements(self, yaml_part: PromptPart):
```

Comment:
weak preference to rename to `_validate_template_data` to be more explicit about what is being validated.